test/com/ableton/JenkinsMocksTest.groovy

```diff
@@ -13,4 +14,11 @@ class JenkinsMocksTest extends BasePipelineTest {
   void setUp() {
     super.setUp()
   }
+
+  @Test
+  void isUnix() throws Exception {
+    // It would be pretty redundant to basically re-implement this method in its own test
+    // case, so instead we just assert the call to see that it didn't go haywire.
```

Comment:
@nre-ableton we don't assert anymore.

Reply:
Sorry, I forgot to fix this in my last round.